GeeksCoreLibrary/Modules/Templates/Services/TemplatesService.cs

```diff
@@ -1163,6 +1166,29 @@ public async Task<JArray> GetJsonResponseFromQueryAsync(QueryTemplate queryTempl
             return result;
         }
 
+        /// <inheritdoc />
+        public async Task<JArray> GetJsonResponseFromRoutineAsync(RoutineTemplate routineTemplate, string encryptionKey = null, bool skipNullValues = false, bool allowValueDecryption = false)
+        {
+            var routineName = $"WISER_{routineTemplate.Name}";
```

Comment:
In overleg met Ferry is er besloten om de "WISER_" prefix die voor routines staat weg te halen. Dit is ook al ontwikkeld in een nieuwe Wiser branch, maar nog niet doorgevoerd. Ik weet dus niet of het slim is om nu nog de aanpassing door te voeren met "WISER_" ervoor, terwijl we over een paar weken het waarschijnlijk weer aan moeten gaan passen.

Dit is dus meer iets om te overleggen, en niet zozeer een change request.

Reply:
Oké, dat wist ik niet. Ik zag dat overal die prefix werd gebruikt, dus had ik dat overgenomen.